### ISSUE TITLE
App displays "low exposure" by default if it encounters an error

### ISSUE THREAD

**Issue by Androp0v** [RadarCOVID/radar-covid-ios#16]
🔑 It seems to me that the app displays by default a "low exposure" label even if it encounters an error while retrieving the DPT3 exposure status and the exposure status is unknown. From ExpositionUseCase.swift:

```
case .inactive(let error):
        var errorEI = ExpositionInfo(level: expositionInfoRepository.getExpositionInfo()?.level ?? .healthy)
        errorEI.error = dp3tTracingErrorToDomain(error)
        return errorEI
```

Also, in this case no value is passed to the lastCheck property of the ExpositionInfo structure, so the app displays a default date of last check "01.07.2020", which is hardcoded in ExpositionViewController:

```
if let date = self.lastCheck {
    let formatter = DateFormatter()
    formatter.dateFormat = "dd.MM.YYYY"
    return formatter.string(from: date)
}
return "01.07.2020"
```

This combination makes the app give the impression that it is working even if it's not. In my opinion the app should default to something like "unknown exposure" in case an error is encountered while retrieving the tracing status from DP3T. At the very least the default date of "01.07.2020" should be changed to "Unknown date of last check" (properly localized of course) to better reflect that the app is not actually working. In the current state the app fails silently which could be seen as misleading.

Would you be open to a PR implementing this?

**Comment by aalmenar** [comment]
This happens on iOS 14 and 13.7 maybe it’s related to #4

**Comment by Androp0v** [comment]
Yes, this happens since iOS 13.7 and iOS 14b6 but should be changed even if/when #4 is fixed in case another issue arises in the future that results in an error when using DPT3. The app should give some kind of warning when it's not working, not default to "low exposure" in case of error.

**Comment by aalmenar** [comment]
Yes, both are related and also the error handling should be better than indicating Low Exposure

**Comment by JuanpaG94** [comment]
Please, this and #4 are **REALLY URGENT** since the app **will not work** on iOS 13.7 and iOS 14(it's release is planned for this month) till this is fixed

**Comment by Androp0v** [comment]
Maybe issues like #35 would be easier to debug if the app didn't default to "low exposure". There's still no way to tell from the app UI if it's working since the app fails silently.

At least the default date of "01.07.2020" is no longer used.

**Comment by daronleaj** [comment]
Actually on the app it shows low when no previous state is found, but when last state is risk or infected it shows the correct state.
